iceberg/iceberg-handler/src/test/java/org/apache/iceberg/mr/hive/TestHiveIcebergStatistics.java

```diff
@@ -351,25 +384,5 @@ private void checkColStatMinMaxDistinctValue(String tableName, String colName, i
     // Check distinct
     Assert.assertEquals("distinct_count", rows.get(5)[0]);
     Assert.assertEquals(String.valueOf(distinct), rows.get(5)[1]);
-
-    shell.executeStatement("set hive.iceberg.stats.source=iceberg");
```

Comment:
is hive.iceberg.stats.source=iceberg tested somewhere else from now?

Reply:
i've added a parametrized config for that, see https://github.com/apache/hive/pull/5648/files/d645b25a042b57895156b38c792e967c598218a3#diff-85f53d8d84a6e67e63a21991e0a2fd0bd243cbb43a7fef18a791324b8c997dc7R52-R66